test/autotest_old.py

```
@@ -0,0 +1,1314 @@
+          os.close(x.stdin)
+        elif x.stdin:
+          x.stdin.close()
+        if isinstance(x.stdout,int):
+          os.close(x.stdout)
+        elif x.stdout:
+          x.stdout.close()
+        if isinstance(x.stderr,int):
+          os.close(x.stderr)
+        elif x.stderr:
+          x.stderr.close()
+      except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)